src/main/java/CodeCatalyst.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Scanner;
+
+public class CodeCatalyst {
```

Comment:
Consider refactoring some code into a new class as this class seems to have multiple purposes.